models/question.js

```diff
@@ -0,0 +1,24 @@
+var mongoose = require('mongoose');
+
+const userSchema = mongoose.Schema({
```

Comment:
@matthewrapp Shouldn't this be the question or trivia Schema?
```suggestion
const triviaSchema = mongoose.Schema({
```

Reply:
I would say questionSchema is more appropriate name